website/flake.nix

```diff
@@ -329,6 +329,8 @@ COMMIT;";
     fi
 
     export HOME=\$TEMP_DIR
+    export XDG_CONFIG_HOME=\$TEMP_DIR/.config
+    export XDG_CACHE_HOME=\$TEMP_DIR/.cache
```

Comment:
Where did this change come from? This flake file is generated by garn, right?

Reply:
This was a change to the `editGarnConfig` `Executable` that we include in `website/garn.ts`. So right now this change also appears on `main` after running `cd website ; garn generate`.